### ISSUE TITLE
Is there a Live ISO?

### ISSUE THREAD

**Issue by probonopd** [eloquentos/system#3]

**Comment by chrisnharvey** [comment]
Not yet. The scripts in here will make an Arch Linux based live ISO, which can be run with SystemImageKit.

**Comment by probonopd** [comment]
Very cool.

**Comment by probonopd** [comment]
I guess a `README.md` is missing...

**Comment by chrisnharvey** [comment]
Yeah, I'm working on a readme that will explain how to build the iso. I'm also trying to get travis to build the iso in a arch chroot environment.

**Comment by probonopd** [comment]
You could use Docker on travis-ci to do that. See https://github.com/probonopd/AppImages/tree/master/recipes/subsurface for an example where I build a Qt app in a CentOS 6 docker container on travis-ci.
